tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -953,10 +1028,11 @@ def _check_traceback(running_coro, exc_type, pattern):
 @cocotb.test()
 def test_exceptions_direct(dut):
     """ Test exception propagation via a direct yield statement """
+
     @cocotb.coroutine
     def raise_inner():
         yield Timer(10)
-        raise ValueError('It is soon now')
+        raise ValueError("It is soon now")
```

Comment:
This change is not reflected in the expected traceback string later in this (and the two other such) test(s).
Easy to change, but to be aware of.